RetroBar/Controls/Clock.xaml.cs

```diff
@@ -108,50 +112,58 @@ private static void SetConverterCultureRecursively(DependencyObject main)
                 {
                     BindingOperations.SetBinding(main, TextBlock.TextProperty,
                         new Binding(binding.Path.Path)
-                            { StringFormat = binding.StringFormat, ConverterCulture = CultureInfo.CurrentCulture });
+                            { StringFormat = binding.StringFormat, ConverterCulture = ci });
                 }
 
                 for (int i = 0; i < VisualTreeHelper.GetChildrenCount(main); i++)
                 {
                     if (VisualTreeHelper.GetChild(main, i) is UIElement sub)
                     {
-                        SetConverterCultureRecursively(sub);
+                        SetConverterCultureRecursively(sub, ci);
                     }
                 }
             }
         }
 
-        private void SetCurrentCulture()
+        private void UpdateUserCulture()
         {
             try
             {
```

Comment:
I think `_userCulture` can be a local variable here instead of above, what do you think?